book/src/overview.md

```diff
@@ -1,19 +0,0 @@
-# Intro
-
-The Chainlink Testing Framework (CTF) is a blockchain development framework written in Go. 
-
-Its primary purpose is to help Chainlink developers create extensive integration, e2e, performance, and chaos tests to ensure the stability of the chainlink project.
-
-It can also be helpful to those who just want to use chainlink oracles in their projects to help test their contracts, or even for those that aren't using chainlink.
-
-This documentation is primarily for:
-- Developers looking to write end-to-end tests
-- Quality Assurance Engineers aiming to test locally
-
-To get started with writing tests, refer to the [Framework](./framework/getting_started.md) chapter, where we guide you from basic to more complex scenarios.
-
-[Repository](https://github.com/smartcontractkit/chainlink-testing-framework) contains two major pieces:
-- [Framework](framework/overview.md)
-- [Libraries](libraries.md)
-
-If you're a non-technical user or want to build integration with Chainlink not in `Golang`, please refer to our [Interactive](framework/interactive.md) chapter.
```

Comment:
"a non-technical user" => "if you don't have full context what's available" WDYT?